requirements-app.txt

```diff
@@ -1,16 +1,16 @@
 # This is used to create Postgres stored functions via the migrations process.
 alembic_utils >= 0.8.2
 
-Authlib >= 1.2.1
-boto3 >= 1.34.7
-botocore >= 1.34.7
-cachelib >= 0.10.2
+Authlib >= 1.3.0
+boto3 >= 1.34.48
+botocore >= 1.34.48
+cachelib >= 0.12.0
 cachetools >= 5.3.2
-celery[sqs] >= 5.3.1
+celery[sqs] >= 5.3.6
 click-datetime >= 0.2
 
 # Datadog Tracer and Profiler
-ddtrace >= 1.18.2
+ddtrace == 1.18.2
```

Comment:
Why is this being pinned to a specific version in this file?  This means it won't get upgraded during the next routine upgrade.  I think `>=` is correct.

Reply:
I was told to handle datadog dependencies in a separate ticket

Reply:
Kyle asked him to pin it, it needs to be pinned.